### PR TITLE
docs: add samples dropdown-menu

### PR DESCRIPTION
**Pull request by Anna-MariiaPetryk**

### CODE REVIEW

**Review by ala-n — APPROVED** [review]

File: pages/src/common/dropdown-menu.less [hunk 1]
```diff
@@ -1,24 +1,45 @@
-.container-dropdown-menu {
-  width: 100%;
-  background-color: #343a40;
+// TODO: review after base styles creation
+// TODO: share with core ?
+.esl-sample-menu {
   display: flex;
+  width: 100%;
   height: 38px;
+  background-color: #343a40;
```
Comment (dshovchko) [inline pages/src/common/dropdown-menu.less:7]: It would be nice to have the color values in separate variables.

**Review by dshovchko — APPROVED** [review]

File: pages/src/localdev.less [hunk 2]
```diff
@@ -1,9 +1,11 @@
 @import (inline) "./libs/bootstrap.css";
 @import "./common/markdown.less";
 @import "./common/page.less";
+@import "./common/dropdown-menu.less";
 @import "./header/logo.less";
 @import "./header/header.less";
 
+
```
Comment (abarmina) [inline pages/src/localdev.less:8]: very small remark - unnecessary space

File: pages/views/samples/menu.njk [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+        <esl-panel class="menu-container"
+                   group="esl-nav"
+                   fallback-duration="400"
+                   close-on-esc close-on-outside-action>
+          <div class="esl-sample-menu-400">{{ lorem.paragraphs(1) }}</div>
+        </esl-panel>
+      </li>
+      <li class="menu-item">
+        <esl-trigger class="menu-item-btn menu-item-arrow">Menu 3
+        </esl-trigger>
```
Comment (abarmina) [inline pages/views/samples/menu.njk:116]: mb can we do it in one line, like in line 106?

**Review by abarmina — APPROVED** [review]

File: pages/src/localdev.less (hunk 2 above)
Comment (ala-n) [reply pages/src/localdev.less:8]: That's ok, it visually separates different import categories

File: pages/src/common/dropdown-menu.less (hunk 1 above)
Comment (ala-n) [reply pages/src/common/dropdown-menu.less:7]: Will be updated in bounds of redesign stories

**Pull request merged**